### PR TITLE
docs: add website announcement bar feature highlight

### PR DESCRIPTION
**Pull request by mcanouil**
This pull request adds a new highlight about the new feature: the website announcement bar. With this feature, users can add a customizable announcement bar to their websites, allowing them to highlight important information, alerts, promotions, or updates. The announcement bar can be positioned above or below the navbar and can be dismissed by the user. Users can also choose from a variety of Bootstrap icons and format the content using markdown.

### CODE REVIEW

**Comment by cderv** [comment]
Isn't it 1.5 feature ? If so, I think we should do the PR on prelease branch for the prerelease website. 

@cwickham do we want to still do highlights for this release or start using prerelease branch ? I would think the latter.

**Comment by cwickham** [comment]
Yeah, I think we want highlights to live on the prerelease site.

For this PR:
* Target the `prerelease` branch
* Move `docs/prerelease/1.5/website-announcement.qmd` to a permanent home. Perhaps `docs/websites/_website-announcement.md` and then put it in `docs/websites/website-tools.qmd` as an `{{< include >}}`?
* Link directly to the permanent home from `_highlights.yml`

**Comment by mcanouil** [comment]
Refactored to include the announcement in "website tools".

**Comment by mcanouil** [comment]
- https://deploy-preview-1104.quarto.org/docs/websites/website-tools.html#announcement-bar

Maybe decrease the width of the screenshot?

**Comment by cderv** [comment]
> Maybe decrease the width of the screenshot?

Or the reduce the height while keep the width ?

**Comment by mcanouil** [comment]
I meant keep the ratio but make it smaller^^

**Comment by cderv** [comment]
> I meant keep the ratio but make it smaller^^

Yes I got it. I am thinking not keeping the ratio, just reduce the height and keep the width ^^

**Comment by mcanouil** [comment]
The, I'll crop the screenshot ;)

File: docs/books/book-output.qmd [hunk 1]
```diff
@@ -34,6 +34,7 @@ Note that in the above configuration the `highlight-style` option applies to all
 HTML books are at their core [Quarto Websites](/docs/websites/website-basics.qmd) with some special navigational behavior built in. This means that all of the features described for enhancing websites are also available for books, including:
 
 -   [Navbars](/docs/websites/website-navigation.qmd#top-navigation)
+-   [Announcement Bar](/docs/websites/website-tools.qmd#announcement-bar)
```
Comment (cwickham) [inline docs/books/book-output.qmd:37]: I doesn't currently work for Books. I've opened an [issue](https://github.com/quarto-dev/quarto-cli/issues/10114). Omit this for now.

File: docs/prerelease/1.5/_highlights.qmd [hunk 2]
```diff
@@ -16,4 +16,6 @@ Quarto 1.5 includes the following new features:
 
 -   [Project Pre Render Scripts](/docs/projects/scripts.qmd#pre-and-post-render)---Project metadata and the render list are now re-computed after any pre-render scripts have executed.
 
--   [Element-wide disabling of HTML table processing](/docs/authoring/tables.qmd#library-authors)---Declare the comment `<!--| quarto-html-table-processing: none -->` anywhere in an HTML RawBlock.
\ No newline at end of file
+-   [Element-wide disabling of HTML table processing](/docs/authoring/tables.qmd#library-authors)---Declare the comment `<!--| quarto-html-table-processing: none -->` anywhere in an HTML RawBlock.
+
+-   [Website Announcement Bar](/docs/websites/website-tools.qmd#announcement-bar)---Add an announcement bar to your website.
```
Comment (cwickham) [inline docs/prerelease/1.5/_highlights.qmd:21]: This looks good. But can you move it up to be under "Website Drafts"?

File: docs/websites/website-tools.qmd [hunk 3]
```diff
@@ -29,6 +29,34 @@ margin-header: |
...
 
+## Announcement Bar
+
+![Captured view of the Quarto website, showcasing the dynamic announcement bar feature.](images/website-announcement.png){.border fig-alt="Screenshot showing an announcement bar on Quarto website with an 'info-circle' icon, a dismiss button, and a primary colour type. The content reads 'Alert - this is some information that you should pay attention to'."}
+
+**Website announcement bar** feature is a versatile tool designed to enhance user engagement and communication on your website.
+By adding a simple configuration to your `_quarto.yml` file, you can display a prominent, customizable announcement bar that grabs visitors' attention.
+This feature allows you to set an icon, make it dismissable, and even include formatted content like bold text.
+It's perfect for highlighting important information, such as alerts, promotions, or updates, directly to your audience.
+The announcement bar can be positioned to fit seamlessly within your site's layout (*e.g.*, `below-navbar` or `above-navbar`), ensuring the message is both impactful and integrated.
```
Comment (cwickham) [inline docs/websites/website-tools.qmd:40]: Some wording/ordering edits to make this a bit more concise:
```suggestion
Add an announcement to display a prominent, customizable bar at the top of your website that grabs visitors' attention.
It's perfect for highlighting important information, such as alerts, promotions, or updates.
You can set an icon, make it dismissable, and even include formatted content like bold text.
The announcement bar can be positioned to fit seamlessly within your site's layout (*e.g.*, `below-navbar` or `above-navbar`), ensuring the message is both impactful and integrated.
```

**Review by cwickham — CHANGES_REQUESTED** [review]
Looking good! Just a few suggestions in the comments.

File: docs/books/book-output.qmd (hunk 1 above)
Comment (mcanouil) [reply docs/books/book-output.qmd:37]: I thought it would.
Nice catch!

Comment (mcanouil) [inline docs/books/book-output.qmd:37]:
```suggestion
```

File: docs/prerelease/1.5/_highlights.qmd (hunk 2 above)
Comment (mcanouil) [reply docs/prerelease/1.5/_highlights.qmd]: done

**Review by cwickham — APPROVED** [review]

**Pull request merged**